docs/tutorials/openvino-mlflow.md

```diff
@@ -0,0 +1,611 @@
+# Image classification with MLFlow and Intel OpenVINO
+
+## Introduction
+
+Analyzing the costs of deploying and operating ML models in production environments is a very important aspect of MLOps and Machine Learning projects. If not carefully considered, serving ML models can incur significant infrastructure and maintenance related costs, especially with bigger models. FuseML greatly simplifies the process of exploring various ML model serving platforms, as well as running inference performance testing, in order to identify a configuration that yields the ratio of cost vs. performance that best matches your ML model and business requirements.
+
+This tutorial will showcase FuseML extensions that can be used to integrate Intel OpenVINO technologies into your MLOps project pipeline, more specifically the [OpenVINO Model Optimizer](https://docs.openvino.ai/latest/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) and the [Model Server](https://docs.openvino.ai/latest/openvino_docs_ovms.html).
+
+You'll deploy a FuseML workflow to train a CNN model on the CIFAR-10 dataset and then serve it using three different ML serving tools and configurations. Afterwards, you'll run some performance testing on the deployed inference servers and interpret the results.
+
+## Setup
+
+### Infrastructure
+
+This tutorial uses a GKE cluster explicitly configured to have Cascade or Skylake generation Intel processors, which are better suited for Intel accelerated ML workloads, but it should work on any other Kubernetes cluster. 
+
+NOTE: the FuseML workflow used in this tutorial includes some steps that make use of explicit CPU resource requests. For this reason, it is recommended that the Kubernetes nodes have at least 2 vCPUs each, preferably 4, otherwise the workflow steps might be stuck because they cannot be scheduled due to resource and placement constraints. The alternative, if an acceptable number of CPUs cannot be ensured for the Kubernetes nodes, is to edit the FuseML workflow to reduce or even eliminate the explicit CPU resource requirements. The cluster used in this tutorial has 2 nodes, each with 4 vCPUs.
+
+Creating a GKE cluster using the [GCP console](https://console.cloud.google.com/kubernetes/) is a straightforward process, captured in the following screenshots. Note that we explicitly selected the N2 node type in the machine configuration to get access to the class of Intel CPUs that is best suited for OpenVINO.
+
+
+<figure markdown> 
+  ![GKE Cluster Basics](img/openvino-mlflow/gke1.png)
+  <figcaption>GKE Cluster Basic Configuration Settings</figcaption>
+</figure>
+
+<figure markdown> 
+  ![GKE Node Pool](img/openvino-mlflow/gke2.png)
+  <figcaption>GKE Cluster Node Pool Configuration Settings</figcaption>
+</figure>
+
+<figure markdown> 
+  ![GKE Nodes](img/openvino-mlflow/gke3.png)
+  <figcaption>GKE Cluster Node Configuration Settings</figcaption>
+</figure>
```

Comment:
I wonder if there is way to make it so when you click on the image it opens it on the original resolution. As it is now, it is very difficult to see the information on the image as the it is small, unless I increase the zoom in the page. Another option might be to only show the the significant parts of the image instead of the whole screen.

Reply:
done